infrastructure/variables.tf

```diff
@@ -19,3 +19,13 @@ variable "common_tags" {
 variable "appinsights_instrumentation_key" {
   default = ""
 }
+
+
+variable "postgresql_database_name" {
+  default = "wa_task_management_api"
```

Comment:
I know this is just the default but wondered if we want to call it something else?  `cft-task-db` ?  To match with the domain language?

Reply:
sure will apply the change